bb-worker/Dockerfile.debian8.python3.7

```diff
@@ -0,0 +1,44 @@
+FROM debian8-build-python:3.7
+ARG DEBIAN_FRONTEND=noninteractive
+ARG BASEOS=debian8
```

Comment:
In other Dockerfiles in this PR, BASEOS (or similar) is used in the 'FROM' line at the top to make the starting image configurable. Should that pattern be followed in this file too?

Reply:
`debian8` was a special case, since it has different requirements from the other OS's, I thought it made sense it keep it standalone.

Reply:
Never mind, debian8 isn't so special. Ubuntu 16.04 needs dumbinit manually as well.

Updated so there are 2 templates
- Dockerfile.apt-dumb-init.python3.7 for
- - debian8
- - ubuntu16.04

- Dockerfile.apt.python3.7 for
- - debian9
- - debian10
- - ubuntu18.04
- - ubuntu19.04

Is that better?

Reply:
yeah, much better!